Formula/breakpad.rb

```diff
@@ -0,0 +1,69 @@
+class Breakpad < Formula
+  desc "Components implement a crash-reporting system."
+  homepage "https://chromium.googlesource.com/breakpad/breakpad/"
+
+  url "https://chromium.googlesource.com/breakpad/breakpad",
+      :using => :git,
+      :revision => "67d5567177301d0c24303f26ad119ab7bd7fab40"
```

Comment:
Google's policy on this when asked previously has tended to be "Pick a commit point that works for you and stick it in your application/project/etc". They don't really provide support for package managers in a conventional way.